modules/openapi-generator/src/main/resources/rust/model.mustache

```diff
@@ -85,23 +90,27 @@ impl Default for {{classname}} {
 {{!-- for non-enum schemas --}}
 {{^isEnum}}
 {{^discriminator}}
+#[serde_as]
```

Comment:
I need some help with this: I don't quite know how to only add this macro if one of the struct attrs. is a byte array.

Reply:
i would do it in post process model: https://github.com/OpenAPITools/openapi-generator/blob/73f1f29eeb19640439fbbd46f19f93dee8d0f91a/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java#L280

and add an extension to the model (e.g. x-rust-has-byte-array) if at least one of the properties is a byte array